ironfish/src/telemetry/telemetry.ts

```diff
@@ -18,7 +18,7 @@ import { Metric } from './interfaces/metric'
 import { Tag } from './interfaces/tag'
 
 export class Telemetry {
-  private readonly FLUSH_INTERVAL = 5 * 60 * 1000
+  private readonly FLUSH_INTERVAL = 1 * 60 * 1000
```

Comment:
@danield9tqh Does this need to be reverted? I'm guessing this was added in debugging due to the message being blocked at the API level

Reply:
Yea thought I did revert that hmm

Reply:
I pushed a revert of this